drivers/gpu/drm/i915/i915_gem_gvtbuffer.c

```diff
@@ -194,8 +194,7 @@ static int gvt_decode_information(struct drm_device *dev,
 		return -EINVAL;
 	}
 
-	args->size = (((args->width * args->height * args->bpp) / 8) +
-		      (PAGE_SIZE - 1)) >> PAGE_SHIFT;
+	args->size = (args->stride * args->height) >> PAGE_SHIFT;
 
```

Comment:
The ALIGN(XX, PAGE_SIZE) should be used firstly and then shifted by using PAGE_SHIFT.

Reply:
Updated, Thanks for the comment